clients/java/src/main/java/io/camunda/client/api/search/enums/UserTaskFilterState.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.client.api.search.enums;
+
+public enum UserTaskFilterState {
```

Comment:
❓ I believe this is simply a `UserTaskState`, isn't it? Do we need to separate this into a `UserTaskFilterState` and a `UserTaskResultState`? The same pattern can be seen for other enums here as well. I believe we can remove the "Filter" and "Result" from the type and state enums here. Let me know if I am missing anything here.

💭 If we create those enums twice, we have to usually keep both in sync while maintaining them. That is really error-prone.